packages/blade/src/utils/platform/getNativePlatform.native.ts

```diff
@@ -0,0 +1,7 @@
+import { Platform } from 'react-native';
+
+export const getNativePlatform = (): typeof Platform.OS => {
```

Comment:
This naming is quite confusing. 

getNativePlatform?? Native means react-Native or Native to the platform...!? 

Can't we use `getPlatformType` util?

Reply:
It's Native to the platform.

`getPlatformType` returns `react-native` but at some places we also want to differentiate between `web`, `ios`, and `android` so that's what it does.

Agree that it is a bit confusing name. Couldn't think of anything else 🤔

Reply:
Oh renamed it to `getOS`. I think that makes more sense. on web it will return `web` but I think that should be fine since that is what react-native-web also seems to do for `Platform.OS`.

Reply:
Sounds good.